ITE/4/README.adoc

```diff
@@ -546,19 +548,20 @@ with artifacts is to record their hashes using one or more cryptographic hash
 algorithms. Other artifact operations such as verifying the artifact rules
 rely on the hashes recorded in the link metadata.
 
-We found that recording hashes of abstract resources is more complicated than the recording hashes of
-artifacts in the local filesystem, particularly if these resources may
-live at remote locations. It is, therefore, important for ITE-4
+We found that recording hashes of abstract resources is more complicated than
+the recording hashes of artifacts in the local filesystem, particularly if these
+resources may live at remote locations. It is, therefore, important for ITE-4
 compliant in-toto tools to handle the resolving of generic URIs to these
-abstract resources in a secure manner. It is quite possible that the contents of these
-resources need to be serialized in a manner that allows for their hashing, and these
-operations, when performed unsafely, can lead to severe vulnerabilities if the
-resource is controlled by a malicious actor. Implementers must also take
-care to ensure ITE-4 compliant systems are capable of handling situations where
-an abstract resource is unavailable, and fail appropriately.
-
-NOTE: There is an ongoing discussion about limiting the scope of this ITE to
-static or unchanging resources. Thread: https://github.com/in-toto/ITE/issues/7
+abstract resources in a secure manner. It is quite possible that the contents of
```

Comment:
Good advice, but how? Where should the non-expert reader turn to for best secure practices?

Reply:
It's very contextual as it depends on each independent type of resource, and the corresponding best practices. I've updated to indicate that, with some general examples.